R/normalize.R

```diff
@@ -122,6 +122,21 @@ sd_check <- function(x) {
     )
     x[zero_sd] <- 1
   }
+
+  na_sd <- which(is.na(x))
+  if (length(na_sd) > 0) {
+    glue_cols <- glue::glue_collapse(
+      glue("`{names(na_sd)}`"), sep = ", ", last = " and "
+    )
+    rlang::warn(
+      glue(
+        "Column(s) {glue_cols} returned NaN, because variance cannot be ",
+        "calculated and scaling cannot be used. ",
+        "Consider avoiding `Inf` or `-Inf` values and/or setting `na_rm=TRUE` ",
+        "before normalizing."
+      )
+    )
```

Comment:
We have been pushing to use the [{cli}](https://cli.r-lib.org/) package recently. Please take a look at the linked issues here https://github.com/tidymodels/recipes/issues/1237 to see how you can convert the `rlang::warn(glue())` to `cli::cli_warn()`

Reply:
Thanks, changed!